plugins/linear-comparative-view/src/LinearComparativeView/model.ts

```diff
@@ -112,6 +114,17 @@ export default function stateModelFactory(pluginManager: PluginManager) {
         )
       },
 
+      // automatically removes session assemblies associated with this view
+      // e.g. read vs ref
+      beforeDestroy() {
+        const session = getSession(self)
+        self.assemblyNames.forEach(name => {
+          if (name.endsWith('-temp')) {
```

Comment:
i think use something more specific than '-temp' in case other views are doing the same thing, so you don't delete their temp assemblies.

actually probably need to use this view's unique id to establish ownership so you don't delete temp assemblies from other instances of LinearComparativeView

Reply:
the -temp concept is created before the view is created so no view.id available, also only looks in the view's assemblies so the "damage" should be minimal